### PR TITLE
Check slice index to avoid 'runtime error: slice bounds out of range' panic

### PR DESCRIPTION
**Pull request by mattmanning**

### CODE REVIEW

**Comment by mattmanning** [comment]
[500s on /services requests](https://trello.com/c/IaH5bK1E/1476-500s-on-services-requests)

**Comment by ddollar** [comment]
Possible to add a quick test around this? There are some good examples in the provider code.

**Comment by mattmanning** [comment]
@ddollar Ready for re-review. Thanks for the testing help.

**Comment by JoshuaGross** [comment]
Thanks for doing this! Glad our wacky rack helped find this bug ;)

**Comment by ddollar** [comment]
Needs a rebase

**Comment by mattmanning** [comment]
Ok rebased and tests passing

File: provider/aws/service_test.go [hunk 1]
```diff
@@ -28,9 +28,47 @@ func TestServiceWebhookURL(t *testing.T) {
 	}
 }
 
+func TestServiceList(t *testing.T) {
```
Comment (ddollar) [inline provider/aws/service_test.go]: This test seems badly named and in the wrong file. Did the rest of these tests get renamed?

**Comment by mattmanning** [comment]
@ddollar Rebased, migrated my new test to resources_test.go, removed services_test.go (which was a duplicate of the new resources_test.go).

**Pull request closed without merging**